### ISSUE TITLE
RasConnectionWatcher events don't fire on windows 11

### ISSUE THREAD

**Issue by Cekdisout** [winnster/DotRas#105]
Hi,
I noticed that RasConnectionWatcher Connected and Disconnected events fires only on first connection in windows 11. 
For all other connections it will not fire again. Maybe anyone else have/spotted same issue?

**Comment by jeff-winn** [comment]
Yeah, definitely seems like the events aren't firing after the first one occurs. I just confirmed it over here as well using the console runner.

**Comment by kp-timo-beyel** [comment]
Any update on the above. Seems it only happens if the connection has been established by the same process. Watching it when the connection gets setup or disconnected from an external process seems to work fine.

**Comment by burakgunvar** [comment]
is there any update on issue. Watcher just triggers only once for both connect & disconnect

**Comment by jeff-winn** [comment]
Well after gutting all of the code to facilitate testing and going straight to the raw APIs to do it, the results were the same.

Given the behavior occurring it very well could be a bug in Windows. It doesn’t appear to be an issue with DotRas nor the .NET code that talks to the Windows CreateEventEx API.

Going to check on my old Windows 10 laptop to confirm it’s not an issue there.

Just gotta remember how to open bug reports in Windows since it’s been quite a while since I reported one last related to this project.

**Comment by jeff-winn** [comment]
Given this is a vendor issue and not something I can address directly, I’ve removed the critical tag so the product can still release 2.0 and will not block it.

The issue will continue to remain open until the problem is solved.

At best, a workaround might be possible in the interim.

**Comment by jeff-winn** [comment]
Confirmed it works fine on Windows 10, it's definitely an issue with Windows 11.
